salt/utils/vmware.py

```diff
@@ -432,9 +431,9 @@ def get_new_service_instance_stub(service_instance, path, ns=None,
         Version of the new stub.
         Default value is None.
     '''
-    #For python 2.7.9 and later, the defaul SSL conext has more strict
-    #connection handshaking rule. We may need turn of the hostname checking
-    #and client side cert verification
+    # For python 2.7.9 and later, the defaul SSL conext has more strict
```

Comment:
`defaul` --> `default`

I know that wasn't really what you're changing in here, but since you've edited the line anyway, might as well fix that up :)

Reply:
I find it hard to disagree on that one. While beautifying, one might as well *read* it too ...